chain-abci/src/main.rs

```diff
@@ -2,17 +2,16 @@ use log::info;
 use std::fs::File;
 use std::net::SocketAddr;
 use std::path::{Path, PathBuf};
-#[cfg(not(feature = "mock-validation"))]
-use zmq::{Context, REQ};
 
 use chain_abci::app::ChainNodeApp;
-#[cfg(feature = "mock-validation")]
+#[cfg(any(feature = "mock-validation", not(target_os = "linux")))]
```

Comment:
Does this mean on mac default `cargo build` will has `mock-validation` semantic.

Reply:
if not linux

Reply:
`any` is or -- so it's either if one explicitly builds with that feature flag, or if one builds on non-Linux OS